src/tools/tool_cmdline.rs

```diff
@@ -38,6 +38,7 @@ struct CmdlineToolConfig {
     #[serde(default = "_default_timeout")]
     timeout: u64,
     background: Option<CmdlineToolBackground>,
+    output_filter: Option<crate::postprocessing::pp_command_output::CmdlineOutputFilter>,
```

Comment:
import it in the top as crate::postprocessing::pp_command_output::CmdlineOutputFilter occurs 3 !!! times in the file

Reply:
I like to see the full path

Reply:
I dislike when this string of 40 chars repeats 3 bloody times in a code